data/crac-io/crac-io-json/src/test/java/com/farao_community/farao/data/crac_io_json/CracImportExportTest.java

```diff
@@ -133,6 +134,25 @@ public void cracTest() {
                 .newOnStateUsageRule().withUsageMethod(UsageMethod.FORCED).withContingency(contingency1Id).withInstant(Instant.CURATIVE).add()
                 .add();
 
+        // network action with one switch pair
+        crac.newNetworkAction().withId("switchPairRaId")
+                .withName("switchPairRaName")
+                .withOperator("RTE")
+                .newSwitchPair().withSwitchToOpen("to-open").withSwitchToClose("to-close", "to-close-name").add()
+                .newFreeToUseUsageRule().withUsageMethod(UsageMethod.AVAILABLE).withInstant(Instant.PREVENTIVE).add()
+                .newOnStateUsageRule().withUsageMethod(UsageMethod.FORCED).withContingency(contingency1Id).withInstant(Instant.CURATIVE).add()
+                .add();
+
+        // network action with two switch pairs
+        crac.newNetworkAction().withId("switchPairRaId2")
+            .withName("switchPairRaName2")
+            .withOperator("RTE")
+            .newSwitchPair().withSwitchToOpen("to-open").withSwitchToClose("to-close", "to-close-name").add()
+            .newSwitchPair().withSwitchToOpen("to-open-2", "to-open-name-2").withSwitchToClose("to-close-2").add()
+            .newFreeToUseUsageRule().withUsageMethod(UsageMethod.AVAILABLE).withInstant(Instant.PREVENTIVE).add()
+            .newOnStateUsageRule().withUsageMethod(UsageMethod.FORCED).withContingency(contingency1Id).withInstant(Instant.CURATIVE).add()
+            .add();
+
```

Comment:
Before doing this, can we start to collect a data-base of "former CRAC json", to ensure that the modifications that we are making does not break the compatibility of the importer with the previous files.

For instance, could you intercept the CRAC JSON of this test before adding these new network actions, and add a unit test on its import. 

I think we should initiate a folder of "former JSON CRACs", and a new TU file which ensure that all the former JSON CRAC can still be imported without crashing.